armi/reactor/converters/uniformMesh.py

```diff
@@ -853,48 +913,13 @@ def _setParamsToUpdate(self, direction):
                     if not name in excludedParamNames
                 ]
             )
+        if direction == "in":
+            self.blockParamNames.extend(["molesHmBOL", "massHmBOL"])
```

Comment:
Should these be added to the parameter categories or why does it make sense to have this here?

Reply:
They're already in the `depletion` category.

1. We could add `depletion` to the categories to be mapped, but then we'd also get a bunch of other parameters that we don't need.
2. We could create a new category `heavyMetalMass` or something along those lines and use it here. But I think we should avoid creating too many parameter categories for very specific use cases.

I'm trying to balance having a reasonable number of categories against having explicit parameter mapping specifications like this. If it was 10 parameters I would create a category, but for two parameters it seems better to just explicitly name them.

Reply:
Could we at least save those strings off as constants at the top of the file?

```python
HEAVY_METAL_PARAMS = ["molesHmBOL", "massHmBOL"]
```

Or some such.